pkg/config/template.go

```diff
@@ -345,6 +345,37 @@ allowed_annotations = [
 # Kata Containers with the Firecracker VMM
 #[crio.runtime.runtimes.kata-fc]
 
+# The workloads table defines ways to customize containers with different resources
+# that work based on annotations, rather than the CRI.
+# Each workload, has a name, activation_annotation, annotation_prefix and set of resources it supports mutating.
+# The currently supported resources are "cpu" (to configure the cpu shares) and "cpuset" to configure the cpuset.
```

Comment:
In slack we said "cpushares" instead of "cpu". I can change the enhancement back to "cpu" if you'd prefer.

Reply:
This needs to be updated to cpushares based on the latest agreement :)